### PR TITLE
Add entrypoints for Simulcast Targets and DeliveryUsage

### PR DESCRIPTION
**Pull request by dylanjha**
- [x] simulcast targets
- [x] delivery usage
- [x] add `yarn format` and `yarn format:fix` commands
- [x] run `yarn format` as part of the travis ci build
- [x] run `yarn format` on this project to fix formatting issues

### CODE REVIEW

**Comment by dylanjha** [comment]
@JacquiManzi what's the correct formatting/linting command? I see we have .eslintrc.json and .prettierrc.js but I want to add a `yarn format` or `yarn lint` command (and also add that as a command for CI)

File: src/video/resources/liveStreams.js [hunk 1]
```diff
@@ -226,6 +226,104 @@ class LiveStreams extends Base {
...
+      );
+    }
+
+    if (!params) {
+      return Promise.reject(
+        new Error(
+          'A url and a stream_key are required to create a simulcast target'
+        )
+      );
+    }
```
Comment (JacquiManzi) [inline src/video/resources/liveStreams.js]: Let's check to make sure these are required- our official docs might just not be indicating that they are.

File: src/video/resources/liveStreams.js [hunk 2]
```diff
@@ -226,6 +226,104 @@ class LiveStreams extends Base {
...
+  /**
+   * Get a simulcast target
+   * @param {string} liveStreamId - The ID for the live stream
+   * @param {string} simulcastTargetId - The ID for the simulcast target to delete
+   * @returns {Promise} - Returns a resolved Promise with a response from the Mux API
+   *
+   * @example
+   * const { Video } = new Mux(accessToken, secret);
+   *
+   * // Create a live stream playback ID
```
Comment (JacquiManzi) [inline src/video/resources/liveStreams.js]: Copy-pastaed comment

File: src/video/resources/liveStreams.js [hunk 3]
```diff
@@ -226,6 +226,104 @@ class LiveStreams extends Base {
...
+   * Video.LiveStreams.getSimulcastTarget(liveStreamId, simulcastTargetId);
+   *
+   * @see https://docs.mux.com/reference#retrieve-a-simulcast-target
+   */
+  getSimulcastTarget(liveStreamId, simulcastTargetId) {
+    if (!liveStreamId) {
+      return Promise.reject(
+        new Error('A Live Stream ID is required to get a simulcast target')
```
Comment (JacquiManzi) [inline src/video/resources/liveStreams.js]: feels odd to capitalize `Live Stream`

**Pull request merged**